doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -29,16 +30,18 @@ Ceph cluster enables the throttling of the operations(IOPS) belonging to
 different client classes (background recovery, scrub, snaptrim, client op,
 osd subop)”*.
 
-The purpose of the mclock profile is to help with the following,
+The purpose of the mclock profile is to help with the following:
 
-- Based on the capacity limits provided and the mclock profile selected,
-  determine the low level mclock resource control parameters,
-- Transparently apply the lower level mclock resource control parameters and
-  some Ceph configuration parameters depending on the profile.
+- the mclock profile uses the capacity limits provided within it and the mclock
+  profile that you select to determine the low-level mclock resource control
+  parameters
+- the mclock profile transparently appies the lower-level mclock resource
```

Comment:
s/appies/applies/

- depending on the profile, lower-level mclock resource control parameters and some Ceph-configuration parameters are transparently applied.

Reply:
note to future me: s/appies/applies/

Reply:
Done.